bot/commands/misc/convert.js

```diff
@@ -0,0 +1,52 @@
+// Converts from one currency to another based on this API: http://exchangeratesapi.io/
+
+const {Command} = require('yuuko');
+const fetch = require('node-fetch');
+const convert = require('convert-units');
+
+function convertUnits (msg, baseValue, baseType, targetType) {
+	const conversion = convert(baseValue).from(baseType).to(targetType);
+	msg.channel.createMessage(`${baseValue}${baseType} is equal to ${conversion}${targetType}`);
```

Comment:
I generally prefer to keep the message sending in the body of the command, just for organization purposes. Consider having this function return the string and sending the message after calling this function. This will let you remove the `msg` argument.